packages/flutter_tools/test/general.shard/build_system/targets/ios_test.dart

```diff
@@ -741,7 +741,10 @@ void main() {
         throwsA(isException.having(
           (Exception exception) => exception.toString(),
           'description',
-          contains('does not contain arm64 armv7. Running lipo -info:\nArchitectures in the fat file:'),
+          contains(
+            'does not contain architectures "arm64 armv7".\n\n'
+            'lipo -info:\nArchitectures in the fat file:',
+          ),
```

Comment:
Here and elsewhere. This is fine since we’re just checking error messages, but we haven’t had an armv7 slice in a few years at this point, so optionally consider cleaning up.

Reply:
Updated!